src/main/java/com/google/sps/app/servlets/NewTripServlet.java

```diff
@@ -66,6 +73,12 @@ public String validateInput(HttpServletRequest request) {
         } catch (NumberFormatException e) {
             return "Invalid totalBudget";
         }
+        if(parseInputDate(request.getParameter(START_DATE_PARAM))==null){
+            return "Invalid date";
```

Comment:
Make the error more specific - "Invalid start date" and "Invalid end date" and "end_date must be >= start_date". If we ever decide to show errors in the frontend, this will be helpful